packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -145,6 +145,35 @@ export function buildRootObject() {
       getBrandForIssuer: issuer =>
         issuerTable.getIssuerRecordByIssuer(issuer).brand,
       getAmountMath,
+
+      makeZCFMint: (keyword, mathHelperName = 'nat') => {
+        const zoeMintP = E(zoeInstanceAdmin).makeZoeMint(
+          keyword,
+          mathHelperName,
+        );
+        const issuerRecordP = E(zoeMintP).getIssuerRecord();
+        const zcfMint = harden({
+          getIssuerRecord: () => {
+            return issuerRecordP;
+          },
+          mintAllocation: (zcfSeat, seatKeyword, amount) => {
```

Comment:
This should take an optional existing seat. If not provided, it returns the new seat.

 `mintAllocation(keywordRecord, toPosition : ZcfSeat?) => ZcfSeat`

Reply:
aside from parameter names, yes.